soc/xtensa/intel_adsp/common/ipc.c

```diff
@@ -84,12 +96,13 @@ int intel_adsp_ipc_init(const struct device *dev)
 	 * the other side!), then enable.
 	 */
 	config->regs->tdr = INTEL_ADSP_IPC_BUSY;
-	if (IS_ENABLED(CONFIG_SOC_INTEL_CAVS_V15)) {
-		config->regs->idd = CAVS_IPC_IDD15_DONE;
-	} else {
-		config->regs->ida = INTEL_ADSP_IPC_DONE;
-		config->regs->tda = INTEL_ADSP_IPC_DONE;
-	}
+#ifdef CONFIG_SOC_INTEL_CAVS_V15
+	config->regs->idd = CAVS_IPC_IDD15_DONE;
+#else
+	config->regs->ida = INTEL_ADSP_IPC_DONE;
+	config->regs->tda = INTEL_ADSP_IPC_DONE;
```

Comment:
This has no effect on ACE. To ACK TDA (to clear BUSY on host side), bit 31 must be cleared on ACE, not set. Just write 0, or use ACK_TDA() as suggested above.